CHANGELOG.md

```diff
@@ -6,6 +6,33 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Latest](https://github.com/NordicSemiconductor/svada)
 
+## [v2.0.0](https://github.com/NordicSemiconductor/svada/tree/v2.0.0)
+
+This version is a large expansion of svada to support a lot of new features.
+```svada``` now supports parsing the whole SVD device in a way mostly following the SVD specification, and more of the content in the SVD file is exposed.
```

Comment:
nit: For Markdown, all of these triple-ticks can be replaced with just single back-ticks.